src/tokens/transformation/android/colorName.js

```diff
@@ -1,11 +1,13 @@
-const camelCase = require('../common/camelCaseHelper')
-
 module.exports = {
   type: 'name',
   matcher: function (token) {
     return token.type === 'color'
   },
   transformer: function (token) {
-    return camelCase(token.path.slice(2).join(' '))
+    return token.name
+    .replace('color_light_','')
+    .replace('color_dark_','')
+    .replace('color_legacy_light_','')
+    .replace('color_legacy_dark_','')
```

Comment:
Does this mean we don't strip the `color_` prefix from `color_primitive` colors?

Reply:
i have kept `color_` prefix only for primitive colors to differentiate in light .xml file.

Reply:
Could we maybe remove the `color_` - we'd still get the `primitive_` prefix on the color to differentiate, which is what we do for the CSS variables